packages/site/src/layout/ContentView.tsx

```diff
@@ -0,0 +1,25 @@
+import { Content } from "@jobber/components";
```

Comment:
I moved `ContentView` into layouts since I think it belongs beside `ComponentView` and `BaseView`

Reply:
Great decision!